karma.mc.config.js

```diff
@@ -89,12 +89,11 @@ module.exports = function(config) {
             use: [{
               loader: "babel-loader", // require("babel-core")
               options: {
+                presets: [["@babel/preset-env", {"targets": {"firefox": "60"}}]],
```

Comment:
What's the motivation for choosing version 60 here?  That probably wants a comment too.

I notice that the presets seem to be duplicated in a several places, perhaps this wants to be hoisted to a single const that's reused in various places?